notifications.bs

```diff
@@ -261,7 +268,7 @@ language tag. Validity or well-formedness are not enforced. [[!LANG]]
 <ol>
   <!-- XXX https://www.w3.org/Bugs/Public/show_bug.cgi?id=24055 -->
   <li><p>If the notification platform supports icons, <a>fetch</a>
-  <var>notification</var>'s <a>icon URL</a>, if <a>icon URL</a> is set.
+  <var>notification</var>'s <a>icon URL</a> if it has been set.
```

Comment:
I'm not sure this change is needed. "Is set" is the wording we use to determine presence. The creation steps also say: "(Otherwise icon URL is not set.)"